Diptych/Diptych/Presentation/Profile/View/ProfileView.swift

```diff
@@ -7,37 +7,94 @@
-            Spacer()
-            Text("ProfileView")
-            Spacer()
-            HStack {
-                Button{
-                    userViewModel.signOut()
-                } label: {
-                    Text("로그아웃")
-                        .padding()
-                        .background(Color.offBlack)
-                        .foregroundColor(.offWhite)
+        ZStack {
+            Color.offWhite.edgesIgnoringSafeArea(.top)
+            
+            VStack {
+                
+                HStack(spacing: 0) {
+                    userNameLabel(text: "쏜야")
+                    Image("heart")
+                        .frame(width: 24, height: 24)
+                        .padding(.horizontal, 60)
+                    userNameLabel(text: "밍니")
+                } // HStack
+                
+                Divider()
+                    .padding(.top, 80) // 임의로 조정했음
+                    .padding(.bottom, 103)
+                
+                Text("D+233")
+                    .font(.pretendard(.light, size: 28))
+                    .foregroundColor(.offBlack)
+                    .padding(.bottom, 30)
+                
+                Text("20번째 딥틱 중")
+                    .font(.pretendard(.light, size: 28))
+                    .foregroundColor(.offBlack)
+                    .padding(.bottom, 179)
+                
+                Text("버전 정보 1.0.0.0")
+                    .font(.pretendard(.light, size: 18))
+                    .foregroundColor(.darkGray)
+                    .padding(.bottom, 22)
+                
+                Text("개인정보처리방침")
+                    .font(.pretendard(.light, size: 18))
+                    .foregroundColor(.darkGray)
+                // == 충돌 부분 시작 ==
+                HStack {
+                    Button{
+                        userViewModel.signOut()
+                    } label: {
+                        Text("로그아웃")
+                            .padding()
+                            .background(Color.offBlack)
+                            .foregroundColor(.offWhite)
+                    }
+                    Button{
+                        userViewModel.deleteAccount()
+                    } label: {
+                        Text("회원탈퇴")
+                            .padding()
+                            .background(Color.offBlack)
+                            .foregroundColor(.offWhite)
+                    }
                 }
-                Button{
-                    userViewModel.deleteAccount()
-                } label: {
-                    Text("회원탈퇴")
-                        .padding()
-                        .background(Color.offBlack)
-                        .foregroundColor(.offWhite)
-                }
-            }
-            Spacer()
-        }
+                // == 충돌 부분 끝 ==
+            } // VStack
+        } // ZStack
     }
 }
 
+// MARK: - Component
+
+extension ProfileView {
+    func userNameLabel(text: String) -> some View {
```

Comment:
method 이름은 동사로 시작하는걸 추천해요 ㅎㅎㅎ
```suggestion
    func makeUserNameLabel(text: String) -> some View {
```